src/i18n/i18n.js

```diff
@@ -213,6 +221,14 @@ const dict = {
         copy_amount: "Betrag kopieren",
         copy_bip21: "BIP21 kopieren",
         copied: "In die Zwischenablage kopiert!",
+        backup_skip: "Überspringen",
+        backup_refund: "Rückerstattungsdatei herunterladen",
+        backup_refund_subline:
+            "Lade deine Rückerstattungsdatei herunter und bewahre sie sicher auf. Du benötigst diese Datei, um deine Bitcoin zurückzuerhalten, falls der Swap fehlschlägt.",
+        backup_refund_list:
+            "are using incognito mode\nare using tor browser\nclear your browsing history",
+        backup_refund_skip:
+            "Du kannst diese Datei später herunterladen, indem du auf den Button in der oberen rechten Ecke klickst.",
```

Comment:
If any of the above applies to you, we strongly recommend downloading this refund file!
```suggestion
            "Falls einer dieser Punkte zutrifft, empfehlen wir ausdrücklich die Rückerstattungsdatei herunterzuladen!",
```